spec/block_spec.rb

```diff
@@ -18,8 +18,8 @@
   end
 
   describe '.previous_transaction' do
-    it 'returns the previous transaction hash' do
```

Comment:
overlap of work on block class

Reply:
As above

Reply:
As above. Just merge now, and replace with Dan's.